src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,6 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream().filter(user -> user.matchName(name)).findFirst().orElse(DEFAULT_USER);
```

Comment:
```suggestion
        return users.stream()
                .filter(user -> user.matchName(name))
                .findFirst()
                .orElse(DEFAULT_USER);
```
한칸씩 내려주면 좀 더 가독성있게 표현해줄 수 있겠네요 :)